plugin/python-imports.vim

```diff
@@ -115,9 +115,18 @@ if !exists("g:pythonBuiltinModules")
 endif
 
 if !exists("g:pythonPaths")
-    let g:pythonPaths = []
+    let g:pythonPathsNorm = []
+else
+  "normalize  paths
+  let _tmp = []
+  for path in g:pythonPaths
+    let path_without_slash = substitute(expand(path), "/$", "", "")
+    call add(_tmp, path_without_slash)
+  endfor
+  let g:pythonPathsNorm = _tmp
```

Comment:
Ah, no,you had it right originally by doing the normalization at the point of use.

If the user changes g:pythonPaths, the next :ImportName should pay attention to that change, which won't happen if you do the normalization once, during plugin load time.